_pages/relayimps/me.untethr.nostr-relay.md

```diff
@@ -0,0 +1,12 @@
+---
+title: me.untethr.nostr-relay
+permalink: 
```

Comment:
These permalinks ... If we don't set them, the path could dictate the link but I'm not sure it's configured. Have you tried running this locally?

Maybe we should remove the permalink line from all files and just default to `/relayimps/me.untethr.nostr-relay` in this case.

Reply:
me.untethr.nostr-relay 
- with permalink - does not show
- with permalink set to /relayimps/me.untethr.nostr-relay/ - does not show

The naming convention of this  relay implementation is not optimal. 
Another option is to just rename it, as we did with /sostr/

Reply:
Please rename it. File and permalink.

Reply:
renamed to untethr.md